django_netjsonconfig/tests/test_admin.py

```diff
@@ -255,6 +255,15 @@ def test_preview_template(self):
         self.assertNotContains(response, 'system')
         self.assertNotContains(response, 'hostname')
 
+    def test_wrong_uuid_field_in_change_404(self):
+        """
+            Device not saved in the databse
+        """
```

Comment:
again I have to ask you to please remove the docstring, it means remove the entire docstring.

Reply:
@nemesisdesign, I was going to do so but you had explicitly mentioned `**this** docstring` in your comment. Hence, left the above one. Nevertheless, won't include any docstrings in further PRs unless a different logic is being used :smiley:

Reply:
as a general rule, follow the conventions used in the files you modify and add comments whenever the code is not easy to read and understand at glance